src/components/button/Button.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+interface ButtonProps {
+    onClick: () => void;
+    label: string;
+    disabled?: boolean;
+}
+
+const Button: React.FC<ButtonProps> = ({ onClick, label, disabled }) => (
```

Comment:
this component is not doing much. its almost an exact wrapper of the native button component